tests/acceptance/pageObjects/filesPage.js

```diff
@@ -335,6 +336,26 @@ module.exports = {
       return this
         .waitForElementVisible('@restorePreviousVersion')
         .click('@restorePreviousVersion')
+    },
+    enterFilterInput: function (input) {
+      return this.initAjaxCounters()
+        .click('@filterListButton')
+        .waitForElementVisible('@filterTextField')
+        .clearValue('@filterTextField')
+        .setValue('@filterTextField', input)
+        .waitForOutstandingAjaxCalls()
+        // This click is for closing the filter list
+        .click('@filterListButton')
+    },
+    getAllFilesListed: async function () {
```

Comment:
This returns all files and folders, not only the files
```suggestion
    getAllListedResources: async function () {
```

Reply:
fixed